DQMOffline/Trigger/plugins/METMonitor.cc

```diff
@@ -0,0 +1,297 @@
+#include "DQMOffline/Trigger/plugins/METMonitor.h"
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "DQM/TrackingMonitor/interface/GetLumi.h"
+
+#include "CommonTools/TriggerUtils/interface/GenericTriggerEventFlag.h"
+
+
+double PI = 3.2;
+int NPI = 64;
+MEbinning phi_binning_{
+  64, -3.2, 3.2
```

Comment:
Neither PI nor NPI are actually used, but probably planned to make the above line of magic numbers look better :)